### DISCUSSION TITLE
Multiple stations under one Icecast port

### DISCUSSION THREAD

**Discussion by radio2019** [AzuraCast/AzuraCast#5040] [Q&A]
Is there a way to have multiple stations running under one Icecast port, so don't:

Station 1: Port 8000
Station 2: Port 8010
Station 3: Port 8020
Station 4: Port 8030

but: Station 1-4 all on port 8000, so that all stations appear on one Icecast page.

I tried to change the port of the station running on 8010 to 8000, but received a message that port 8000 was already in use by another station.
Any fix available for that?

**Comment by BusterNeece** [comment]
We don't support that officially in the application, since we try to keep each station "sandboxed" into its own independent frontend and backend combination that isn't affected by others; there are unofficial ways to do this, namely by setting the secondary stations up, then having them "relay" their signal back to the main Icecast server as a secondary output.

**Comment by radio2019** [comment]
Thank you.
I tried that, but it didn't work, although it should: [https://prnt.sc/26lfi0q](https://prnt.sc/26lfi0q)
When I click on [https://rsra.net:8000/grooves.mp3](https://rsra.net:8000/grooves.mp3) I always get a "The file you requested could not be found" message.
The AutoDJ is activated as you can see here: [https://prnt.sc/26lfk4u](https://prnt.sc/26lfk4u) & [https://prnt.sc/26lfkwe](https://prnt.sc/26lfkwe)
What I have done wrong?

**Comment by radio2019** [comment]
OK, now it works: [https://s20.directupload.net/images/220129/jjebsm8p.png](https://s20.directupload.net/images/220129/jjebsm8p.png)
